src/commands/deployments/editScmType.ts

```diff
@@ -8,13 +8,18 @@ import { DeploymentsTreeItem } from "vscode-azureappservice";
 import { IActionContext } from "vscode-azureextensionui";
 import { ScmType } from "../../constants";
 import { SiteTreeItem } from "../../explorer/SiteTreeItem";
+import { TrialAppTreeItem } from '../../explorer/trialApp/TrialAppTreeItem';
 import { WebAppTreeItem } from "../../explorer/WebAppTreeItem";
 import { ext } from "../../extensionVariables";
+import { localize } from '../../localize';
 
 export async function editScmType(context: IActionContext, node?: SiteTreeItem | DeploymentsTreeItem, newScmType?: ScmType, showToast?: boolean): Promise<void> {
     if (!node) {
         node = <SiteTreeItem>await ext.tree.showTreeItemPicker(WebAppTreeItem.contextValue, context);
     } else if (node instanceof DeploymentsTreeItem) {
+        if (node.parent instanceof TrialAppTreeItem) {
+            throw Error(localize('notSupported', 'This operation is not supported.'));
```

Comment:
Think it makes sense to just add this string to the constants since it gets used three times in the repo.

Reply:
Would suggest following the "errors.ts" pattern instead of using "constants.ts". For example: https://github.com/microsoft/vscode-azuretools/blob/master/ui/src/errors.ts